person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] Exactly the same thing here ⬆, A method for adding rentals should be present in this class `Person` and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association 💯

> For Example 💡

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```